src/components/NavBar/NavBar.js

```diff
@@ -5,35 +5,43 @@ import { Link } from 'react-router-dom'
 import HamburgerMenu from '../HamburgerMenu/HamburgerMenu'
 import './NavBar.scss'
 
-const NavBar = () => {
+const NavBar = ({ setUser }) => {
   const [mobileNavOpen, setMobileNavOpen] = useState(false)
 
   const active = classNames('mobile-menu', {
     open: mobileNavOpen,
   });
 
+  const loginUser = () => {
+    // use graphql to fetch user
+    // set state to that user object
+  }
+
   return (
     <>
     <nav className="navigation-wrapper">
-      <h1>Troubadour</h1>
       <div className="navigation-header">
         <div className="navigation-names">
           <Link className="link" to="/">
             <img className="logo" src={logo} alt="Crown of instruments"/> 
           </Link>
         </div>
-        <div className="navigation-links">
-          <button
-            aria-label="Toggle Mobile Menu Button"
-            className={active}
-            onClick={() => {
-              setMobileNavOpen((mobileNavOpen) => !mobileNavOpen)
-            }}
-            >
-            <div className="bar-one" />
-            <div className="bar-two" />
-            <div className="bar-three" />
-          </button>
+        <h1>Troubadour</h1>
+        <div className="button-menu-container">
+          <button className="login-button">Admin Login</button>
```

Comment:
Thank you for adding an Admin login. It's not set-up, but will be a part of our product!